addons/website_sale/models/product_template.py

```diff
@@ -401,9 +401,15 @@ def _search_render_results(self, fetch_fields, mapping, icon, limit):
             if with_image:
                 data['image_url'] = '/web/image/product.template/%s/image_128' % data['id']
             if with_category and product.public_categ_ids:
-                data['category'] = _('Category: %s', product.public_categ_ids.name)
-                slugs = [slug(category) for category in product.public_categ_ids]
-                data['category_url'] = '/shop/category/%s' % ','.join(slugs)
+                if len(product.public_categ_ids) == 1:
+                    data['category'] = _('Category: %s', product.public_categ_ids.name)
+                    data['category_url'] = '/shop/category/%s' % slug(product.public_categ_ids)
+                else:
+                    data['category'] = {'extra_link_title': _('Categories: ')}
+                    data['category_url'] = dict()
+                    for categ in product.public_categ_ids:
+                        data['category'][slug(categ)] = _('%s', categ.name)
```

Comment:
Is `_('%s', categ.name)` different from `categ.name` ?

Reply:
Just an oversight :) Thanks !